### PR TITLE
Add file dropzone functionality for SVG drag and drop

### PR DESCRIPTION
**Pull request by thomas-franz**
Hey there! 👋🏼 
Thanks for creating and maintaining @yoksel  :)
Been using this tool for many years now and finally wanted to contribute something to it.

This feature allows you to simply drag and drop SVGs into the tool.

Let me know if theres anything I should adjust/change.

### CODE REVIEW

File: src/scss/styles.scss [hunk 1]
```diff
@@ -281,3 +281,29 @@ DD {
   font-size: .9em;
   color: #777;
 }
+
+#dropzone {
```
Comment (yoksel) [inline src/scss/styles.scss:285]: Use classes for stylization, please. They allows to use modifiers like in other classes here. 
With modifier selectors will look like `.dropzone` and `.dropzone--active`

**Review by yoksel — CHANGES_REQUESTED** [review]

**Comment by yoksel** [comment]
Cool, very convenient!
Tiny comment to code

**Comment by thomas-franz** [comment]
Hey @yoksel , did the changes and it should be ready now.
Please let me know if there's anything else to change.

**Comment by yoksel** [comment]
Thank you!

**Pull request merged**